esx_service/vmdk_ops.py

```diff
@@ -211,6 +220,13 @@ def validate_vsan_policy_name(policy_name, vmdk_path):
     if not vsan_policy.policy_exists(policy_name):
         raise ValidationError('Policy {0} does not exist'.format(policy_name))
 
+def validate_disk_allocation_format(format):
+    """
+    Ensure format is valid.
+    """
+    if not format.lower() in kv.VALID_ALLOCATION_FORMATS :
+        raise ValidationError('Disk Allocation Format {0} does not exist. Valid options are: (thin(default), eagerzeroedthick, zeroedthick)'.format(format))
```

Comment:
I think this statement is way beyond 80char limit or whatever PEP8 recommends. You may want split & indent in 2 lines.